packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -192,6 +193,14 @@ export class EmrCreateCluster extends sfn.TaskStateBase {
     this._serviceRole = this.props.serviceRole ?? this.createServiceRole();
     this._clusterRole = this.props.clusterRole ?? this.createClusterRole();
 
+    // Service role must be able to iam:PassRole on the cluster role
+    this._clusterRole.grantPassRole(this._serviceRole);
```

Comment:
`AmazonEMRServicePolicy_v2` only grants `iam:PassRole` on the default role. Need this here for the cluster role we create.